hasher-matcher-actioner/webapp/package.json

```diff
@@ -57,6 +61,7 @@
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "prettier": "^2.2.1"
+    "prettier": "2.3.1",
```

Comment:
typescript requires a higher version of prettier

Reply:
nit: might was well leave the `^`